windowing.py

```diff
@@ -50,20 +50,20 @@ def find_lift_windows(windowed_data, divisor, threshold, return_lift_windows):
                  Then calculates variance threshold and returns an np array of
                  the windows that meet the variance threshold.
     Args:
-        windowed_data -- np array [[[[x1],[y1],[z1],[nanos1]], [[x2], [y2], 
+        windowed_data -- np array [[[[x1],[y1],[z1],[nanos1]], [[x2], [y2],
                                          [z2], [nanos2]]...]
         divisor -- int (will use divisor to calculate variance threshold so
```

Comment:
Should each argument's comment start capitalized? If you're following an existing convention so maybe it's fine to keep it as is.